### ISSUE TITLE
Verify Flash after Programming

### ISSUE THREAD

**Issue by mikrocoder** [ElTangas/jtag2updi#17]
Hello,

does your "programmer" do a real reading check after writing? "Verify Flash after Programming.
Or does he just write without any check?

**Comment by ElTangas** [comment]
My programmer does whatever avrdude tells it to do. If avrdude requests a verify, then a verify will be done.

Why do you ask? Did you a find a verify bug?

**Comment by mikrocoder** [comment]
no, no bug. I just wanted to know if he was doing a real reading check. My code adaptation for an ATmega328PB is not ready for testing yet. That's why I ask before. Because the Python tool "pyupdi" can not do that.
